packages/panna-sdk/src/core/onramp/onramp.ts

```diff
@@ -194,9 +194,16 @@ export function getOnrampProviders(countryCode: string): ProviderInfo[] {
     throw new Error(`Invalid country code: ${countryCode}`);
   }
 
-  const providers: OnrampProvider[] =
-    COUNTRY_PROVIDER_MAP[normalizedCountryCode] || [];
-  return providers.map((provider) => PROVIDERS[provider]);
+  const mappedProviders = COUNTRY_PROVIDER_MAP[normalizedCountryCode] || [];
+
+  if (mappedProviders.length === 0) {
+    return [];
+  }
```

Comment:
Is the early return intentional? Should countries without any mapped providers still have access to onramp.money, or is onramp.money only meant as an additional option for countries that already have at least one provider?

Current behavior:
- countries with providers -> get their mapped providers + onramp.money
- countries without providers -> get empty array (also no onramp.money)